main.go

```diff
@@ -68,12 +72,18 @@ func (m *GreenReviewsTooling) BenchmarkPipelineTest(ctx context.Context,
 		}
 	}
 
+	if prometheus_url == "" {
+		log.Printf("Missing prometheus url from makefile", err)
+		return nil, err
```

Comment:
In this we can directly use fmt.Errorf as it actually handles the printing as well
and we can avoid this log.Printf even in dagger run this message gets hidden